### PR TITLE
org.StudioPond.Spiderbro

### PR DESCRIPTION
**Pull request by pforpond**
Changes to be committed:
	new file:   data.zip
	new file:   org.StudioPond.Spiderbro.desktop
	new file:   org.StudioPond.Spiderbro.svg
	new file:   polybius-i (spiderbro flatpak).bas
	new file:   polybius_linux
	new file:   polydata.zip
	new file:   spiderbro_linux
	new file:   spiderbroflatpak.yml
	new file:   vame (spiderbro flatpak).bas

Spiderbro is a retro inspired adventure puzzle game. You play as Spiderbro, a purple spider that has to face an oppressive and dystopian environment whilst overcoming the obstacles of a shadowy adversary.

- [X ] Please describe your application briefly.
- [X ] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X ] My pull request follows the instructions at [App Submission][submission].
- [ X] I have [built][build] and tested the submission locally.
- [X ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [ X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X ] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [ X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: polybius_linux
Comment (hfiguiere) [inline polybius_linux:1]: this does not belong here.

File: polydata.zip
Comment (hfiguiere) [inline polydata.zip:1]: this does not belong here.

File: spiderbro_linux
Comment (hfiguiere) [inline spiderbro_linux:1]: this does not belong here.

File: spiderbroflatpak.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+app-id: org.StudioPond.Spiderbro
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```
Comment (hfiguiere) [inline spiderbroflatpak.yml:3]: current runtime is 23.08. 22.08 wil be EOL in August.

File: data.zip
Comment (hfiguiere) [inline data.zip:1]: this does not belong here.

File: vame (spiderbro flatpak).bas
Comment (hfiguiere) [inline vame (spiderbro flatpak).bas:1]: this file is never referenced. and likely this does not belong here.

**Review by hfiguiere — CHANGES_REQUESTED** [review]
there is no appstream file (it's mandatory)

File: spiderbroflatpak.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+app-id: org.StudioPond.Spiderbro
```
Comment (hfiguiere) [inline spiderbroflatpak.yml:1]: `studiopond.org` does NOT exist. Yet

> [ X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

File: spiderbroflatpak.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: spiderbro_linux
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
```
Comment (hfiguiere) [inline spiderbroflatpak.yml:9]: missing `--share=ipc` that is necessary for X11.

File: spiderbroflatpak.yml (hunk 2 above)
Comment (pforpond) [reply spiderbroflatpak.yml:1]: Hi! I own studiopond.co.uk. Would co.uk.studiopond.spiderbro be acceptable or should I just go the io.github route? many thanks!

**Comment by hfiguiere** [comment]
also how does this work on aarch64? I don't see any source code being built.

**Comment by pforpond** [comment]
> also how does this work on aarch64? I don't see any source code being built.

it doesn't work on arm

**Comment by hfiguiere** [comment]
still this would (try to) build an aarch64 package.

File: spiderbroflatpak.yml
Comment (hfiguiere) [inline spiderbroflatpak.yml:1]: the manifest name MUST conform to the app-id as per the documentation. So this file WILL need to be renamed.

File: spiderbroflatpak.yml (hunk 2 above)
Comment (hfiguiere) [reply spiderbroflatpak.yml:1]: whatever you prefer. Just don't make up domains.

**Comment by bbhtt** [comment]
Undo the last merge commit. You merged in all the changes from the master branch of this repo.

**Pull request closed without merging**